packages/theme/src/cli/services/init.ts

```diff
@@ -81,21 +84,78 @@ export async function createAIInstructions(themeRoot: string, aiInstruction: AII
             destination: tempDir,
             shallow: true,
           })
-          const aiSrcDir = joinPath(tempDir, 'ai', aiInstruction)
-
-          let aiDestDir = themeRoot
 
-          if (aiInstruction !== 'claude') {
-            aiDestDir = joinPath(themeRoot, `.${aiInstruction}`)
-          }
+          const instructions =
+            aiInstruction === 'all'
+              ? (Object.keys(SUPPORTED_AI_INSTRUCTIONS).filter((key) => key !== 'all') as AIInstruction[])
+              : [aiInstruction]
```

Comment:
```suggestion
          const instructions = (
            aiInstruction === 'all'
              ? (Object.keys(SUPPORTED_AI_INSTRUCTIONS).filter((key) => key !== 'all') as AIInstruction[])
              : [aiInstruction]
          ) as Exclude<AIInstruction, 'all'>[]
```
I saw a lot of casting happening in code further down the chain, and it could be resolved if we just casted it here instead.